src/globals/icons/svg/group.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
the other svgs don't have `xmlns` but adding it allows the svg preview in github
![image](https://user-images.githubusercontent.com/1596764/49300306-dbcdd300-f48f-11e8-918c-cceda99bc621.png)
 (and others, like my vscode plugin) to work. I can take it out but it is technically more correct

Reply:
Good find! As long as `npm run build:icons` runs correctly happy to go with this and we can correct the others as part of a separate PR.